webapp/src/main/webapp/src/app/shared/base.service.ts

```diff
@@ -25,4 +25,12 @@ export abstract class BaseService<T extends BaseEntity> {
 		return this.httpClient.delete<T>(environment.apiBaseUrl + resource + '/' + entity.id);
 	}
 
+	postCountAndItems(resource: string, data: any): Observable<BaseCountAndItems<T>> {
+		return this.httpClient.post<BaseCountAndItems<T>>(environment.apiBaseUrl + resource, data);
+	}
+
+	getWithParams(resource: string, paramsData: any): Observable<BaseCountAndItems<T>> {
```

Comment:
I think this should be named **searchCountAndItems** and please don't use `any` as a type for method arguments unless a function is truly generic, this should take the same parameter type as the ng resource method you are calling below

Reply:
Done